Sources/JobsPostgres/PostgresJobsQueue.swift

```diff
@@ -13,11 +13,11 @@
 //===----------------------------------------------------------------------===//
 
```

Comment:
I suppose JobPostgres will be deprecated once the `swift-jobs-postgres` repository is ready?

Reply:
I’m just going to delete it from here.